src/main/resources/liquibase/202204041547-add-billing-account-id-column-to-snapshot-table.xml

```diff
@@ -0,0 +1,14 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202204041547-1" author="ksynvrit">
+        <comment>Add column for billing account id to snapshot table</comment>
+        <addColumn tableName="tally_snapshots">
+            <column name="billing_account_id" type="VARCHAR(255)" />
+        </addColumn>
+    </changeSet>
```

Comment:
This should have default `_ANY`, and have a non-null constraint. See `src/main/resources/liquibase/202006180903-add-usage-to-host.xml` for example.

Reply:
Resolved